sdks/python/apache_beam/transforms/ptransform_test.py

```diff
@@ -536,12 +538,23 @@ def test_flatten_no_pcollections(self):
     assert_that(result, equal_to([]))
     pipeline.run()
 
+  @attr('ValidatesRunner')
+  def test_flatten_one_single_pcollection(self):
+    pipeline = TestPipeline()
+    input = [0, 1, 2, 3]
+    pcoll = pipeline | 'Input' >> beam.Create(input)
+    result = (pcoll,)| 'Single Flatten' >> beam.Flatten()
+    assert_that(result, equal_to(input))
+    pipeline.run()
+
+  @attr('ValidatesRunner')
   def test_flatten_same_pcollections(self):
     pipeline = TestPipeline()
     pc = pipeline | beam.Create(['a', 'b'])
     assert_that((pc, pc, pc) | beam.Flatten(), equal_to(['a', 'b'] * 3))
     pipeline.run()
 
+  @attr('ValidatesRunner')
```

Comment:
This one doesn't need to be ValidatesRunner as it's an SDK feature.

Reply:
Reverted.